### DISCUSSION TITLE
i18n in admin management panel

### DISCUSSION THREAD

**Discussion by joyshmitz** [riclolsen/json-scada#13] [Q&A] [answered]
in the admin management panel, the i18n module is provided?

**Comment by riclolsen** [comment, marked as answer]
No it is not. Vuetify has its own internationalization scheme I think, I haven't even studied it yet. I am new to this framework.

**Reply by joyshmitz** [reply]
at first glance, the plugin "Vue I18n is internationalization plugin for Vue.js" https://github.com/kazupon/vue-i18n

**Reply by riclolsen** [reply]
Yes, here is a guide on  how to use https://www.freecodecamp.org/news/how-to-add-internationalization-to-a-vue-application-d9cfdcabb03b/
I will leave this feature for the big Vuetify 3.0 update that will probably require some code adaptation.

**Reply by joyshmitz** [reply]
cool

**Comment by riclolsen** [comment]
Now i18n for admin UI is available. The tags panel is under development, not completed yet.
C:\json-scada\src\htdocs-admin\src\i18n\i18n-current.js

**Reply by joyshmitz** [reply]
thanks for the answer and clarification
